Gruntfile.js

```diff
@@ -11,10 +11,10 @@ module.exports = function (grunt) {
 
     grunt.initConfig({
         pkg,
-        //Executing this task will populate grunt.config.gitinfo with repository data below
-        gitinfo: {
-            local: { branch: { current: { SHA: "", name: "", currentUser: "", } } },
-            remote: { origin: { url: "" } }
+        githash: {
```

Comment:
Does this task do anything?

Reply:
githash/gitinfo used to retrieve information about the current git status such as branch, hash, etc. it is used when we deploy to our local branch. this task used in `grunt deploy-branch`

to use the task we have to configure in the `initConfig` and register the task in the `grunt.registerTask`